fe/fe-core/src/main/java/com/starrocks/catalog/FileTable.java

```diff
@@ -76,6 +77,8 @@ private void validate(Map<String, String> properties) throws DdlException {
         if (!format.equalsIgnoreCase("parquet") && !format.equalsIgnoreCase("orc")) {
             throw new DdlException("not supported format: " + format);
         }
+        // Put path into fileProperties, so that we can get storage account in AzureStorageCloudConfiguration
+        fileProperties.put(AzureCloudConfigurationFactory.AZURE_PATH_KEY, path);
```

Comment:
there are any other properties? just path?

Reply:
For the external file table and broker load, we need to put the file path into the same key, so that we can handle the path easily in AzureStorageCloudConfiguration.